# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/test/copypaste.js
# Commit: 366245552cae
# Full Hash: 366245552cae536ef883be806322395dc47b759d
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-06-17 09:24:05
# Regressor Bug: 1951038
# File Overlap Count: 1
# Description:
#   Bug 1951038 - Get rid of the legacy white-space normalizer of `HTMLEditor` and the pref r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D253589
# ==============================================================================

diff -r b13b0a530402 -r 366245552cae dom/base/test/copypaste.js
--- a/dom/base/test/copypaste.js	Tue Jun 17 00:10:27 2025 +0000
+++ b/dom/base/test/copypaste.js	Tue Jun 17 00:37:22 2025 +0000
@@ -380,11 +380,7 @@
       // The <ul> should appear because it has a <li>
       // The preceding linefeed of the <br> in the empty <li> is an invisible
       // white-space.  Thus, it's not important whether it appears or not in the result.
-      SpecialPowers.getBoolPref(
-        "editor.white_space_normalization.blink_compatible"
-      )
-        ? '<ul id="ul3"><li><br></li></ul>Copy3then Paste'
-        : '<ul id="ul3"><li>\n<br></li></ul>Copy3then Paste'
+      '<ul id="ul3"><li><br></li></ul>Copy3then Paste'
     );
 
     // with elements of different depth